servarr/README.md

```diff
@@ -0,0 +1,116 @@
+> Follow the table below and forget everything else. 
+
+> [!CAUTION] 
+> Please, do not remove Anchors when you see them (the strage syntax with the `&`)
+
+---
+
+## Values
+
+### Jellyfin
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| dash.mail | string | No default value | Insert Jellyfin login mail (will be used also for Jellyseerr integration) |
+| dash.password | string | No default value | Insert Jellyfin password (will be used also for Jellyseerr) |
+| dash.username | string | No default value | Insert the Jellyfin username (will be used also for Jellyseerr) |
+
+### Global
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| global.apikey | string | No default value is configured for security reasons | Insert your Prowlarr, Sonarr, Radarr API key here (one to rule them all!).  Do not remove the `&apikey` anchor! |
+| global.certManagerClusterIssuer | string | No default value, leave empty if not required | Insert your cert manager cluster issuer, e.g.: letsencrypt-cloudflare.  Do not remove the `&issuer` anchor! |
+| global.storageClassName | string | `"network-block"` | Insert your storage class here, e.g.: &storageClassName network-block.  Do not remove the `&storageClassName` anchor! |
+
+### Prowlarr
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| indexers | list | The body of the 1337x index is provided as default | The indexers list. Each element of the list is the yaml-formatted boody of the [Prowlarr API request](https://prowlarr.com/docs/api/#/Indexer/post_api_v1_indexer) to add that index. |
+
+### Optional
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| issuer | object | See the sub fields | For tracking purpose, not used - replaced with pre-existing cluster issuer |
+| issuer.cloudFlareKey | string | `nil` | Insert your CloudFlare key |
+| issuer.email | string | `nil` | Insert your email address |
+
+### Required
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| metrics.enabled | bool | `false` | Anchor to set wether to deploy the export sidecar pods or not. Requires the Prometheus stack.  Do not remove the `&metricsEnabled` anchor! |
+| volumes.downloads.name | string | `"downloads-volume"` | Name of the download pvc.  Do not remove the `&downloads-volume` anchor! |
+| volumes.downloads.size | string | `"100Gi"` | Size of the downloads volume, in Kubernets format |
+| volumes.media.name | string | `"media-volume"` | Name of the media pvc.  Do not remove the `&media-volume` anchor! |
+| volumes.media.size | string | `"250Gi"` | Size of the media volume, in Kubernets format |
+| volumes.torrentConfig.name | string | `"torrent-config"` | Name of the torrent configuration pvc.  Do not remove the `&torrentConfig` anchor! |
+| volumes.torrentConfig.size | string | `"250Mi"` | Size of the torrent configuration volume, in Kubernets format |
+
```

Comment:
What if we move optional and required values in the other categories? Then we can add in the description if it's an optional value or not.

As an example: when reading about the storage parameters is easily to get lost

https://github.com/fonzdm/servarr/blob/58cdaeff0b8aa07f3d9d80c186a821828914b32a/servarr/README.md?plain=1#L106-L113

For the default value we have `"See the sub fields" but we are at the end of the page, and the subfields must be found first 😄

Reply:
Yes, I will remove Required and Optional values... I completely forgot them 😡

Reply:
Done!